theforeman.org/scripts/debian/configure_pbuilder.sh

```diff
@@ -4,29 +4,33 @@ set -xe
 # This script sets up the PBuilder hooks to pull in our deb repositories while the package is building
 echo "--Apt Hook Setup"
 
-for pbuilderd in /etc/pbuilder/${os}*; do
-  # Cleanup any old hooks
-  sudo rm -f ${pbuilderd}/hooks/F60addforemanrepo
+# Cleanup any old hooks
+sudo rm -f /etc/pbuilder/${os}64/hooks/F60addforemanrepo
```

Comment:
that's interesting. currently, we allow different `deb_build_*` jobs to run in parallel (just not the same job), but that means they actually can and break each other if timed "correctly"